rustls/src/crypto/ring/quic.rs

```diff
@@ -252,12 +233,16 @@ mod tests {
         let expander = TLS13_CHACHA20_POLY1305_SHA256_INTERNAL
             .hkdf_provider
             .expander_for_okm(&OkmBlock::new(SECRET));
-        let hpk =
-            super::HeaderProtectionKey::new(expander.as_ref(), version, &aead::quic::CHACHA20);
+        let header_key =
+            hkdf_expand_label_aead_key(expander.as_ref(), 32, version.header_key_label(), &[]);
+        let packet_key =
+            hkdf_expand_label_aead_key(expander.as_ref(), 32, version.packet_key_label(), &[]);
```

Comment:
I think you could knock out the magic 32 constant with a binding that takes advantage of the new `aead_key_len` fn on the trait like:
```
let key_len = TLS13_CHACHA20_POLY1305_SHA256_INTERNAL
            .quic
            .unwrap()
            .aead_key_len();
```